README.md

```diff
@@ -118,6 +373,28 @@ In order to run the account-service in development mode without containers, you
     npm run start:worker:debug
     ```
 
+**Linting:**
+
+```sh
+  npm run lint
+```
+
+**Auto-format:**
+
+```sh
+  npm run format
+```
+
+### Deployment
+
+You can deploy this project using:
+
+Example:
+
+```sh
+
+```
```

Comment:
Yeah, this is TBD. Production deployment would likely be in someone's own cloud environment; maybe at some point we'll have sample Helm charts and such, but honestly it will depend a lot on their specific environment, so even those would just be a starting point. For now, should we mark "TBD", or just remove this section until we have content?